Upstream-Status: Inappropriate [configuration]

proftpd tries to get the IP address from the hostname.
Unluckily now the hostname is not properly configured in /etc/hosts.
We can use this patch as a workaround.

Author: Dexuan Cui <dexuan.cui@intel.com>
Tue Oct 25 12:59:27 CST 2011

--- proftpd-1.3.3c.orig/sample-configurations/basic.conf
+++ proftpd-1.3.3c/sample-configurations/basic.conf
@@ -7,6 +7,9 @@
 ServerType			standalone
 DefaultServer			on
 
+#By default we bind to all interfaces.
+DefaultAddress      0.0.0.0
+
 # Port 21 is the standard FTP port.
 Port				21
 
